internal/signer/linux/pkcs11/pkcs11.go

```diff
@@ -115,18 +121,23 @@ func Cred(pkcs11Module string, slotUint32Str string, label string, userPin strin
 		privKey: privKey,
 		label:   label,
 		module:  *module,
+		// SHA256 initialized as default
+		hash:      crypto.SHA256,
```

Comment:
Instead, create a defaultHash variable and initialize it to SHA256, so your code is self documenting.

Reply:
Should I initialize this variable inside the Cred function?

Reply:
Yes, line 116 should work